build/split-css.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable */
```

Comment:
Is this file used by the css vars plugin?
Also probably better lived somewhere else. `build/` is often used synchronously with `dist/`.

Reply:
this is leftover from testing and can be removed I tried to have postcss split the var definitions into a separate file so that we could have just a scss vars file to include in scss projects but postcss is geared to only produce a single file